sklearn/neighbors/base.py

```diff
@@ -20,11 +22,10 @@
 from ..utils import check_X_y, check_array, _get_n_jobs, gen_even_slices
 from ..utils.fixes import argpartition
 from ..utils.validation import DataConversionWarning
```

Comment:
You missed DataConversionWarning here, that should now be imported from exceptions